scripts/development/m3_stack/m3dbnode.yml

```diff
@@ -3,6 +3,12 @@ coordinator:
     type: "config"
     value: "0.0.0.0:7201"
 
+local:
```

Comment:
hm...I don't think this is needed. m3_stack runs m3coordinator as an external binary, so I don't . think you need to set this here. To be honest I'm not sure why the coordinator config is here at all

Reply:
Hmm I'll leave it here for now.

Reply:
but its not even the correct list of namespaces, `m3_stack` configures a bunch of other namespaces

Reply:
So you think I should remove?

Reply:
I think so, it'll be less confusing. We should probably remove the coordinator config entirely from this file but its a bigger change.